.kitchen.yml

```diff
@@ -35,6 +35,9 @@ suites:
     - recipe[nrpe::default]
     - recipe[nagios_test::default]
     - role[monitoring]
+    verifier:
+      inspec_tests:
+        - test/smoke/server_package
```

Comment:
It's the recommended directory structure for inspec smoke tests when using test-kitchen is under `test/smoke/SUITE_NAME`, and specifying it like that is from the default chefdk generator cookbook so figured I'd keep that up.